content/YubiHSM2/Commands/Sign_Pss.adoc

```diff
@@ -29,7 +29,7 @@ M := Hash link:../Concepts/Algorithms.adoc[Algorithm] to use for MGF1
 
 S := Salt len (2 bytes)
 
-D := H (2 + L bytes, where L is the length of the hashed data)
+D := L bytes of hashed data)
```

Comment:
There's an extra closing parantheses here

Reply:
and we don't define L here, maybe we should try to describe that D is the hashed data, 20, 32, 48 or 64 bytes?

Reply:
Fixed